Formula/wash.rb

```diff
@@ -0,0 +1,19 @@
+class Wash < Formula
+  version = "0.1.0"
+  homepage "https://puppetlabs.github.io/wash"
+  url "https://github.com/puppetlabs/wash/archive/#{version}.tar.gz"
+  sha256 "aadc78ce7209c553c510b609b40c1fd6071360b9df7ba06449095b1f110b46de"
+
+  head "https://github.com/puppetlabs/wash.git"
+
+  depends_on "go" => :build
```

Comment:
Is there a way to restrict the `Go` version to 1.12+?

Reply:
Yeah, I'll add it.

Reply:
Done.